### ISSUE TITLE
the results of training are black pictures

### ISSUE THREAD

**Issue by Junoin** [noahzn/Lite-Mono#147]
Nice work！
I encountered a problem during training. Using the given parameters for training on the kitti dataset, the loss dropped from 0.14 to 0.11, but the depth map I got was a completely black picture or a few black blocks. Does anyone know how to solve this problem?

**Comment by noahzn** [comment]
Hi, how did you visualize your depth maps? Did you check it in the tensorboard?

**Comment by Junoin** [comment]
> Hi, how did you visualize your depth maps? Did you check it in the tensorboard?

Yes. The disp_0  disp_1 disp_2  in the tensorboard are all black pictures. And I also ran the test_simple.py using the weights I trained, but the results are alse very terrible :(

**Comment by noahzn** [comment]
Were they black from the beginning or they suddenly became black during the training? Did you change anything in the code, for example the intrinsics?

**Comment by Junoin** [comment]
> Were they black from the beginning or they suddenly became black during the training? Did you change anything in the code, for example the intrinsics?

Thank you very much for your tips. I checked the parameters and found that the input dimensions of the pre-trained model and the model did not match. After the modification, the displayed disp is the depth map. Thank you again for sharing!

**Comment by noahzn** [comment]
Good to know!!
